python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -82,8 +83,12 @@ def start_ganga():
 
     # [PollThread]autostart_monThreads=False has turned this off being done automatically.
     # The thread pool is emptied by _ganga_run_exitfuncs
-    from Ganga.Core.MonitoringComponent.Local_GangaMC_Service import _makeThreadPool
-    _makeThreadPool()
+    from Ganga.Core.MonitoringComponent.Local_GangaMC_Service import ThreadPool, _makeThreadPool
+    if len(ThreadPool) > 0:
+        del ThreadPool[:]
+        ThreadPool = []
```

Comment:
I have a feeling that this line will perform a name rebinding, not reassign the module level variable to point to something else. It shouldn't be needed anyway if the previous line is doing its job.